### PR TITLE
PMM-5799 : releasing pmm2 client docker

### PR DESCRIPTION
**Pull request by AlyHKafoury**

### CODE REVIEW

File: pmm/pmm2-server-release.groovy [hunk 1]
```diff
@@ -185,13 +185,23 @@ pipeline {
                         docker tag \${DOCKER_CLIENT_VERSION} perconalab/pmm-client:\${VERSION}
                         docker tag \${DOCKER_CLIENT_VERSION} perconalab/pmm-client:\${DOCKER_MID}
                         docker tag \${DOCKER_CLIENT_VERSION} perconalab/pmm-client:\${TOP_VER}
```
Comment (EvgeniyPatlan) [inline pmm/pmm2-server-release.groovy:187]: my opinion - we should release into percona only

**Review by EvgeniyPatlan — CHANGES_REQUESTED** [review]

File: pmm/pmm2-server-release.groovy [hunk 2]
```diff
@@ -182,17 +182,17 @@ pipeline {
...
-                        docker tag \${DOCKER_CLIENT_VERSION} perconalab/pmm-client:\${DOCKER_MID}
-                        docker tag \${DOCKER_CLIENT_VERSION} perconalab/pmm-client:\${TOP_VER}
+                        docker tag \${DOCKER_CLIENT_VERSION} percona/pmm-client:\${VERSION}
+                        docker tag \${DOCKER_CLIENT_VERSION} percona/pmm-client:\${DOCKER_MID}
+                        docker tag \${DOCKER_CLIENT_VERSION} percona/pmm-client:\${TOP_VER}
                         if [ \${TOP_VER} = 1 ]; then
-                            docker tag \${DOCKER_CLIENT_VERSION} perconalab/pmm-client:latest
-                            docker push perconalab/pmm-client:latest
+                            docker tag \${DOCKER_CLIENT_VERSION} percona/pmm-client:latest
+                            docker push percona/pmm-client:latest
```
Comment (EvgeniyPatlan) [inline pmm/pmm2-server-release.groovy:190]: are we going to issue pmm-client for version 1 to percona account too?

**Review by EvgeniyPatlan — CHANGES_REQUESTED** [review]

File: pmm/pmm2-server-release.groovy (hunk 2 above)
Comment (rnovikovP) [reply pmm/pmm2-server-release.groovy:190]: If I remember correctly, we don't have a docker for `percona/pmm-client`  at all . 
So `:latest` will be  -  adding the new one. 

pls, not introduce it right now. There will be a task in Jan. to switch `percona/pmm-server:latest` to  `pmm-server:2` . And at the same time, we'll introduce `:latest`  for pmm-client.


As for now, we need `:2`  ,  `:2.12` and `2.12.0` (as examples ) tags only. The same way we do them for pmm-server

**Review by EvgeniyPatlan — APPROVED** [review]

File: pmm/pmm2-server-release.groovy (hunk 1 above)
Comment (AlekSi) [reply pmm/pmm2-server-release.groovy]: Why?

Comment (AlekSi) [reply pmm/pmm2-server-release.groovy]: Ok, that's fine

**Review by AlekSi — APPROVED** [review]

**Review by hors — APPROVED** [review]

**Pull request merged**